package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "got",
-	"version": "13.0.0",
+	"version": "13.1.0",
```

Comment:
This should be done by the maintainers.

Reply:
Got it. Would be nice to have a CONTRIBUTING.md with mentions what and where we should do. I'll remove this.

Reply:
I don't see the need to document basic contribution things in every project. Generally, you should try to touch the least amount of code and config.

Reply:
Got it. Thanks.